source/guides/network/wifi.rst

```diff
@@ -102,8 +104,119 @@ TUI (Text-based User Interface)
 
    .. figure:: /_figures/wifi/nmtui_5.png
 
+wpa_supplicant
+**************
+
+Make sure NetworkManager is either disabled or not installed before using
+wpa_supplicant.
+
+.. code-block:: bash
+
+   sudo systemctl stop NetworkManager.service
+
+#. Create a ``wpa_supplicant`` configuration directory.
+
+   .. code-block:: bash
+
+      sudo mkdir /etc/wpa_supplicant
+
+#. Create a minimal :file:`/etc/wpa_supplicant/wpa_supplicant.conf` and add the
+   following:
+
+   .. code-block:: bash
+
+      ctrl_interface=/run/wpa_supplicant
+      update_config=1
+
+#. Determine your wireless interface name.
+
+   .. code-block:: bash
+
+      iw dev
+
+#. Complete the configuration process as ``root``.
+
+   .. code-block:: bash
+
+      sudo su
+      wpa_supplicant -B -i $INTERFACE -c /etc/wpa_supplicant/wpa_supplicant.conf
```

Comment:
```suggestion
      sudo wpa_supplicant -B -i $INTERFACE -c /etc/wpa_supplicant/wpa_supplicant.conf
```

sudo to have access to save configuration under `/etc`

Reply:
Actually you have to be logged in as root - mentioned above.  "sudo" did not work for me.

Reply:
If you add `ctrl_interface_group=wheel` to the .conf file it should work with sudo. That would be better than having to switch to root.